engine/apps/oss_installation/views/cloud_users.py

```diff
@@ -38,8 +38,8 @@ def get(self, request):
         organization = request.user.organization
 
         queryset = User.objects.filter(
+            User.build_permissions_query(RBACPermission.Permissions.NOTIFICATIONS_READ, organization),
```

Comment:
**NOTE**: `User.build_permissions_query` now returns `django.db.models.Q` object(s) instead of a dict